### PR TITLE
Add a config yml with groups of repos

### PR DESCRIPTION
**Pull request by agrare**
Add a config file which groups the various repos into buckets which can be used to run operations on e.g. all providers or just core

One use-case is `@miq-bot cross_repo_tests` https://github.com/ManageIQ/miq_bot/pull/526

### CODE REVIEW

File: config/repos.groups.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  manageiq-providers-openshift:
+  manageiq-providers-openstack:
+  manageiq-providers-ovirt:
+  manageiq-providers-redfish:
+  manageiq-providers-scvmm:
+  manageiq-providers-vmware:
+core: &core
+  manageiq:
+  manageiq-api:
+  manageiq-ui-classic:
```
Comment (Fryguy) [inline config/repos.groups.yml:26]: Can you please also add most of the following?

```yaml
  manageiq-automation_engine:
  manageiq-consumption:
  manageiq-content:
  manageiq-decorators:
  manageiq-gems-pending:
  manageiq-graphql:
  manageiq-schema:
  manageiq-ui-service:
  manageiq-v2v:
```

Although ui-service seems to fail, so maybe we should not put that one?

**Comment by Fryguy** [comment]
In hindsight, maybe we should name this file repos.sets.yml (since the option is called `--repo-set` through the code here?).  Not a big deal either way.

**Comment by agrare** [comment]
Done, are there any other groups that might be useful?  I think `providers`, `ui-classic + api`, and `all` would be the most common sets that we want to test against.

**Comment by Fryguy** [comment]
Yeah the ones here seems pretty good to start...if the UI team wants one for them, we can add this later.

**Pull request merged**